src/swaps/hop/HopSwapProvider.ts

```diff
@@ -195,11 +195,26 @@ class HopSwapProvider extends SwapProvider {
     }
   }
 
+  _formatFee(fee: any, networkName: string, type: string ){
```

Comment:
type pls. It should be something like `number | EIP1559Fee` 
`EIP1559Fee` can probably be imported from chainify

Reply:
I've tried it EIP1559Fee, but it is throwing an error saying something like 

"number cannot have property maxFeePerGas"

So looks like these arbitary types number I EIP1559Fee causes some problems on compilation. 

Maybe I am missing something but it behaves like this on my side

Reply:
Maybe in the next version we should think of a way to handle eip1559Fee and Regular gasPrice in some more clever way, because passing arbitrary types is causing warning on TS side

Reply:
Let me DM you see if we can figure it out